strip-test-selectors.js

```diff
@@ -8,34 +8,24 @@ function isTestSelector(attribute) {
   return TEST_SELECTOR_PREFIX.test(attribute);
 }
 
-function StripTestSelectorsTransform() {
-  this.syntax = null;
-}
-
-StripTestSelectorsTransform.prototype.transform = function(ast) {
-  let walker = new this.syntax.Walker();
-
-  walker.visit(ast, function(node) {
-    if (node.type === 'ElementNode') {
-      node.attributes = node.attributes.filter(function(attribute) {
-        return !isTestSelector(attribute.name);
-      });
-    } else if (node.type === 'MustacheStatement' || node.type === 'BlockStatement') {
-      if ('sexpr' in node) {
-        node = node.sexpr;
-      }
```

Comment:
Is this not necessary anymore? I don't know what it does anyway though :)

Reply:
ah, nevermind – fc27e649659cf0df98989934ecb9cb395dffa52e

Reply:
it was only necessary for compat with Ember 1.11/1.12, but we haven't supported that version for quite some time now